File: PHP/7 kyu/Shortest Word.php
<?php
/*
 

DESCRIPTION  

Simple, given a string of words, return the length of the shortest word(s).

String will never be empty and you do not need to account for different data types.



*/

function findShort($str){
  $arr = explode(' ',$str);
  $len = [];
  foreach($arr as $r){
    $len[] = strlen($r);
  }
  return min($len);
}

File: Python/7 kyu/Shortest Word.py

 """ 
Simple, given a string of words, return the length of the shortest word(s).

String will never be empty and you do not need to account for different data types.

 """ 

def find_short(s):
    listRes = list(s.split(" "))
    l = len(min(listRes,key=len))
    return l # l: shortest word length